libs/core/langchain_core/runnables/graph.py

```diff
@@ -11,11 +12,20 @@
     from langchain_core.runnables.base import Runnable as RunnableType
```

Comment:
unrelated but why the rename

Reply:
yea because I need to import the actual class inside a function